### PR TITLE
[MAR- 3221] Add insight charts

### PR DESCRIPTION
**Pull request by malmas18**

### CODE REVIEW

File: apps/marketplace/components/Insights/insights.scss [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    border-color: #666666;
+    border-width: 2px;
+    border-style: solid;
+    text-align: center;
+    height: 200px;
+    width: 100%;
+    margin: 1rem 0rem;
+    padding: 2rem;
+}
+.blue-box > .big-text {
```
Comment (tobiasbrown) [inline apps/marketplace/components/Insights/insights.scss]:
```suggestion

.blue-box > .big-text {
```

File: apps/marketplace/components/Insights/insights.scss [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    position: relative;
+    height: 12rem;
+}
+
+.chart-md-height-2x {
+    position: relative;
+    height: 14rem;
+}
+
+.chart-md-height-3x {
```
Comment (tobiasbrown) [inline apps/marketplace/components/Insights/insights.scss]: This style and some others aren't used.  Do we need them?

File: apps/marketplace/main.scss [hunk 3]
```diff
@@ -10,6 +10,14 @@
   margin-bottom: 3rem;
 }
 
+.margin-bottom-4x {
```
Comment (tobiasbrown) [inline apps/marketplace/main.scss]: This style and some others aren't used.  Do we need them?

File: apps/marketplace/components/Insights/insights.scss (hunk 2 above)
Comment (malmas18) [reply apps/marketplace/components/Insights/insights.scss]: added so we don't have gaps

File: apps/marketplace/main.scss (hunk 3 above)
Comment (malmas18) [reply apps/marketplace/main.scss]: added so we don't have gaps

File: apps/marketplace/components/Insights/DailyRates.js [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+    }
+    let counts = this.props.insightData.briefResponseData.dailyRates
+    counts = counts.filter(v => !['Not Specified', 'Emerging technologies'].includes(v.briefCategory))
+    const chart = new Chart(this.chartRef.current, {
+      type: 'horizontalBar',
+      data: {
+        datasets: [
+          {
+            yAxisID: 'no-show',
+            label: '',
```
[the remark this answers is not in the document]
Comment (malmas18) [reply apps/marketplace/components/Insights/DailyRates.js]: it does really mean 25th to median... this doesn't really matter though as it is not shown

[the remark this answers is not in the document]
Comment (malmas18) [reply apps/marketplace/components/Insights/DailyRates.js]: i mean, it doesn't really mean 25th to median

File: apps/marketplace/components/Insights/DailyRates.js [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+            data: counts.map(a => parseInt(a.briefResponseDayRate25PC, 10))
+          },
+          {
+            label: '25th percentile to median',
```
[the remark this answers is not in the document]
Comment (malmas18) [reply apps/marketplace/components/Insights/DailyRates.js]: it does really mean 25th to median... this doesn't really matter though as it is not shown

**Pull request merged**